DesignSafe-CI/portal
# bug/DES-2048: Remove department from profile

## Overview: ##

The `departments` endpoint in TAS has been deprecated.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2048](https://jira.tacc.utexas.edu/browse/DES-2048)

## Summary of Changes: ##
- Remove references to `department` or `departmentId` from the profile or forms

## Testing Steps: ##
1. Go to https://designsafe.dev/account/profile/edit/ and click "Edit profile", and then "Save"
2. Ensure that your profile updates successfully, and verify changes on reload
